src/ORM.php

```diff
@@ -105,7 +105,7 @@ public function resolveRole($entity): string
     {
         if (is_object($entity)) {
             $node = $this->getHeap()->get($entity);
-            if (!is_null($node)) {
+            if ($node !== null) {
```

Comment:
@wolfy-j FTR, If this change is to improve performance, then it's not necessary since PHP 7.0, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4015#issuecomment-568508507.

Reply:
I guess if it does not make much different I would stick to !== null as easier to read option. This got highlighted by EA extended while I was working on deeper optimizations. Thx for the comment.

Reply:
I can provide ref about a decision in EA extended: https://github.com/kalessil/phpinspectionsea/issues/1486

Reply:
> This got highlighted by EA extended

https://github.com/kalessil/phpinspectionsea/issues/1496 :)

Reply:
;(

Reply:
In any scenario, this wasn't the point of these optimizations, just some extra cleanup.

Reply:
FYI, There is an extra 10% speed improvement and memory reduction we can do.